### PR TITLE
[10.0] Add web_float_empty

### PR DESCRIPTION
**Pull request by yasmineelmrini**

### CODE REVIEW

File: web_float_empty/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Web Float Empty',
+    'version': '10.0.1.1.0',
+    'author': 'Savoir-faire Linux',
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'license': 'LGPL-3',
+    'category': 'Calendar',
```
Comment (rimbendhaou) [inline web_float_empty/__manifest__.py]: @yasmineelmrini are you sure about the category?

File: web_float_empty/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Web Float Empty',
```
Comment (rimbendhaou) [inline web_float_empty/__manifest__.py]: @yasmineelmrini Empty Web Float or Web Float Empty?

Comment (rimbendhaou) [reply web_float_empty/__manifest__.py]: @yasmineelmrini Web Float Field Empty

File: web_float_empty/static/src/js/web_float_empty.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    form_widgets.FieldFloat.include({
+        set_value: function(value_) {
+            /*
+                Override the original function to avoid setting
+                a default value 0 on floats that have the option 
+                no_default_value set to 1.
+            */
+            if (this.options.no_default_value === 1){
+                this.node.attrs.placeholder = '0,00';
```
Comment (joaoalf) [inline web_float_empty/static/src/js/web_float_empty.js]: @yasmineelmrini Ideally we should use the proper language decimal separator.

**Review by joaoalf — CHANGES_REQUESTED** [review]
Just some minor changes.

File: web_float_empty/static/src/js/web_float_empty.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    form_widgets.FieldFloat.include({
+        set_value: function(value_) {
+            /*
+                Override the original function to avoid setting
+                a default value 0 on floats that have the option 
+                no_default_value set to 1.
+            */
+            if (this.options.no_default_value === 1){
+                var precision = this.digits[1];
+                this.node.attrs.placeholder = '0,' + '0'.repeat(precision);
```
Comment (joaoalf) [inline web_float_empty/static/src/js/web_float_empty.js]: @yasmineelmrini I think my point about the decimal separator still stands. Some countries use `,` while others use `.`. Also, what happens if precision == 0?

**Review by joaoalf — CHANGES_REQUESTED** [review]
My page was outdated but my point still stands.

**Review by joaoalf — APPROVED** [review]
lgtm

**Review by rimbendhaou — APPROVED** [review]

**Comment by rimbendhaou** [comment]
ok

**Pull request merged**